code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -914,17 +914,33 @@
 	..()
 	. = 1
 
-/datum/reagent/medicine/stimulants
```

Comment:
Wait, did you just remove normal stimulants from the game?

Reply:
lol accident ill fix that